i18n/messages-ja.xtb

```diff
@@ -1009,4 +1009,29 @@
+	<translation id="6137042847012351902" key="MSG_PODLIST_PODCARDLIST_2" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Title of a pod status column">Status</translation>
+	<translation id="7297543912422139896" key="MSG_PODLIST_PODCARDLIST_3" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Title of a restarts column for a pod">Restarts</translation>
+	<translation id="5517860411754358001" key="MSG_PODLIST_PODCARDLIST_4" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Title of a column for the age of a pod">Age</translation>
+	<translation id="7098139662512349704" key="MSG_PODLIST_PODCARDLIST_5" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Title of a column">Cluster IP</translation>
+	<translation id="7141576551732668109" key="MSG_PODLIST_PODCARDLIST_6" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Title of a column">CPU (cores)</translation>
+	<translation id="2198933707202852928" key="MSG_PODLIST_PODCARDLIST_7" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Title of a column">Memory (bytes)</translation>
+	<translation id="653475364042320864" key="MSG_PODLIST_PODCARDLIST_8" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Tooltip for failed pod card icon">This pod has errors.</translation>
+	<translation id="8022917849813090234" key="MSG_PODLIST_PODCARDLIST_9" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Tooltip for pending pod card icon">This pod is in a pending state.</translation>
+	<translation id="7339906504794526992" key="MSG_PODLIST_PODCARDLIST_10" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Label for logs tooltip">Logs</translation>
+	<translation id="403630050812245993" key="MSG_PODLIST_PODCARDLIST_11" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Name of the pod resource">Pod</translation>
+	<translation id="6710159274365919294" key="MSG_PODLIST_PODCARDLIST_12" source="/usr/local/google/home/bryk/src/github.com/kubernetes/dashboard/.tmp/messages_for_extraction/podlist/podcardlist.html.js" desc="Name of the pod resource">Pod</translation>
```

Comment:
OK, just this problem: now that the variables are named differently, the existing japanese translations are not picked up. So someone will need to move them manually in the `messages-ja.xtb` file to the new variables and delete the old ones (which are still there with the old names). E.g. `MSG_ACTION_BAR_CREATE_ACTION_TOOLTIP` is now `MSG_COMMON_COMPONENTS_ACTIONBAR_ACTIONBAR_0`.